src/libs/actions/App.js

```diff
@@ -61,15 +60,8 @@ function setSidebarLoaded() {
 
     Onyx.set(ONYXKEYS.IS_SIDEBAR_LOADED, true);
     Timing.end(CONST.TIMING.SIDEBAR_LOADED);
-
-    if (!canCapturePerformanceMetrics()) {
-        return;
-    }
-
-    const performance = require('react-native-performance').default;
-    performance.mark('sidebarLoadEnd');
-    performance.measure('timeToInteractive', 'nativeLaunchStart', 'sidebarLoadEnd');
-    printPerformanceMetrics();
+    markEnd(CONST.TIMING.SIDEBAR_LOADED);
+    markStart(CONST.TIMING.SWITCH_REPORT);
```

Comment:
This is the first time a chat loads 
There's an end mark inside <ReportActionsView> 
If there's no "start" mark when the "endMark" is added we'll get an error 
So it's easier and useful to add a "start" mark here and avoid problems

Reply:
Should we perhaps just use a try/catch and ignore this error? 

One thought is that `switch_report` is meant to time the interaction where we are on one report and then switch to another - but in this case we are adding a sample that measures the time to render the first report when the app starts up. Probably won't make a huge difference in practice, but feels a bit out of place here.

Reply:
The `try/catch` would have to be placed in the `PerformanceObserver` as that's where we intercept `_end` marks and make the measure, and it would be necessary only because of this one call

Also if we remove the mark here we won't see the initial report render (to layout) on the timeline 

I think we should capture the initial report rendering the easiest thing would be to rename `TIMING.SWITCH_REPORT` to `TIMING.SHOW_REPORT`? Or add new label in `TIMING` related to initial report appearing?

Reply:
Creating a separate timing mark for the initial report load sounds good. 👍